src/components/ScopeTree/ScopeTree.tsx

```diff
@@ -31,6 +31,20 @@ type ScopeTreeProps = {
   restrictToPractitionerPerimeter?: boolean
 }
 
+const orderRowsByAccess = (rowA: ScopeTreeRow, rowB: ScopeTreeRow): number => {
+  if (rowA.access == rowB.access) {
+    const nameA = rowA.name
+    const nameB = rowB.name
+
+    if (nameA < nameB) return -1
+    if (nameA > nameB) return 1
+  } else if (rowA.access) {
+    return -1
+  }
+
+  return 0
```

Comment:
Shouldn't you return 1 (case where rowB.access is true and rowA.access is false)?

And maybe you could return 0 after line 40